src/test/java/teammates/sqllogic/core/AccountsLogicTest.java

```diff
@@ -68,7 +70,8 @@ public void testDeleteAccount_accountExists_success() {
     }
 
     @Test
-    public void testDeleteAccountCascade_googleIdExists_success() {
+    public void testDeleteAccountCascade_googleIdExists_success()
+            throws InvalidParametersException, EntityAlreadyExistsException {
```

Comment:
Didn't need this previously, seems like an unnecessary change to me.

Reply:
Likely means lint not working properly for this files. We need to fix this in another PR.